v3/docs/gs/about.md

```diff
@@ -71,3 +73,12 @@ Releases of the HSE library and associated language bindings use
 [semantic versioning](https://semver.org).
 Releases of forked repos use a versioning scheme that is documented
 in their associated `README.md`.
+
+The HSE storage format, API, CLI, configuration parameters, and REST interface may change between
+major releases.  However, we strive to minimize changes to the programming API so that existing
+HSE applications require little to no porting effort.
+If the HSE storage format changes, HSE application data must be dumped and reloaded using
+application-specific tools in order to use the new version of HSE.
```

Comment:
For CLI, were were planning on adding `--json` option to emit data in structured form, and that would remain stable between releases.  But the plain text (default) form should not be counted on to remain stable.

Reply:
I don't see anything in the text that implies the CLI text output will be stable between releases, but I'm open to ideas.